source/release-notes/v1.7-release-notes.rst

```diff
@@ -79,9 +105,135 @@ Passenger has been upgraded from ``5.3.7`` to ``6.0.4``. NGINX has been upgraded
+
+Show account balance warnings
+.............................
+
+The dashboard can now show account balance warnings if the users' account balance
+is some threshold.
+
+See :ref:`these docs <balance-warnings-on-dashboard>` for more details on how to 
+enable this feature.
+
+Job Composer and Active Jobs match Dashboard Branding
+.....................................................
+
+The Job Composer and Active Jobs apps' top navigation bar now share the same branding
+scheme configured for the dashboard. 
+
+There's no additional configuration required for this other than what's documented 
+:ref:`here <branding>`.
+
+Maintenance mode
+................
+
+OOD can be put into 'maintenance mode' where it serves a static page to either all
+users or a subset (staff on a VPN for example).  This static page informs users
+that maintenance is underway while administrators perform disruptive tasks.
+
+See :ref:`these docs <maintenance-mode>` on how to configure and enable this feature.
+
+Titles for Files menu entries
+.............................
+
+The shortcuts in the files dropdown menu can now show an optional title along with the 
+directory path. The :ref:`original documentation <add-shortcuts-to-files-menu>` on this 
+feature has been updated to reflect the new options.
+
+
+Documentation for Keycloak as identity broker with CILogon and Keycloak with Duo 2FA
+....................................................................................
+
+Documentation for Keycloak as identity broker with CILogon has been added in the documentation
+entitled ':ref:`authentication-tutorial-oidc-keycloak-rhel7-configure-cilogon`'.
+
+Documentation for Keycloak with Duo two factor authentication has been updated in the documetnation
+entitled ':ref:`authentication-duo-2fa-with-keycloak`'.
+
+Add Grafana support for ActiveJobs app
+......................................
+
+OnDemand's ActiveJobs app can display graphs for jobs that are pulled from Grafana.
+Details on how to configure Grafana support are in :ref:`these docs <grafana-support>`.
+
+Improvements to ood-portal-generator
+....................................
+
+The ood-portal-generator app has been rewritten in Ruby and extensive testing added.
+Checksums generated for ``ood-portal.conf`` now exclude comments.
+
+Add an ssh wrapper to the shell app
+...................................
+
+An ssh wrapper script can now be used instead of just the ``ssh`` command in the
+shell app.  See how to configure this :ref:`here <ssh-wrapper>`.
+
+Regenerate ood-portal.conf with every apache restart
+....................................................
+
+Systemd will now regenerate the ood-portal.conf before every apache restart.  This means
+and administrators can now edit the ``ood_portal.yml`` configuration and then restart httpd
+with systemctl and apache will read the new configuration, because a new ood-portal.conf
+will have been written just before restarting.
```

Comment:
Clarify that

1. this is made possible due to the use of the checksum feature and that this will not work if the checksum indicates that there have been manual changes made to the generated ood-portal.conf
2. clarify that this means when editing `ood_portal.yml` you do not need to follow up with executing ood-portal-generator script
3. clarify the exception is when installing 1.7, where you still have to execute the script with the force flag

Reply:
The checksum feature was introduced in 1.6, it's not new for 1.7.  So I feel like #1 is old news.  Still want #1 called out?

Reply:
Yes, I think my point is that we are leveraging that feature to support safely updating the ood-portal.conf file automatically on every Apache restart.